src/.pre-commit-config.yaml.jinja

```
@@ -120,6 +120,7 @@ repos:
     hooks:
       - id: oca-checks-odoo-module
       - id: oca-checks-po
+        files: '/i18n/[a-zA-Z0-9_]*\.(po|pot)$'
```

IMHO we shouldn't define a particular regex to filter the file because we could miss a case

e.g. `module-name.pot`

Notice the dash `-` for this file
 
What about?

```suggestion
        files: '/i18n/*\.(po|pot)$'
```

---

But a module name cannot contain a dash?

---

@sbidoul 

Oh, you are right, as usual :)


@etobella 

IMHO it should be in the hooks repo:
  - https://github.com/OCA/odoo-pre-commit-hooks/blob/5e97e13778f2cf7440bc711171f9e4db69a0f6a7/.pre-commit-hooks.yaml#L24

---

Well, I made the fixing PR in both repos.

IMO is cleaner to edit the precommit, as it only affects a little set of repositories with documentation